### PR TITLE
Telemetry

### PR DESCRIPTION
**Pull request by blinktag**
* Scaffold telemetry service
* Pass DemoMode flag

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ type Config struct {
 	NumTailWorkers       int              `help:"Number of tail workers to run" default:"4"`
 	NumBroadcastWorkers  int              `help:"Number of broadcast workers to run" default:"4"`
 	DemoMode             bool             `help:"Run server in demo mode. This disables modifications" default:"false"`
+	TelemetryEnabled     bool             `help:"Enable sending usage telemetry to Streamdal" default:"true"`
```
Comment (dselans) [inline config/config.go:33]: Should be opt-in by default so `TelemetryDisabled`.

File: services/telemetry/telemetry.go [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+		case <-t.ShutdownCtx.Done():
+			t.log.Info("stopping telemetry worker")
+			quit = true
+			looper.Quit()
+			return nil
+		case work := <-t.work:
+			batch = append(batch, work)
+			t.log.WithField("data", work).Info("received telemetry data")
+
+			if err := t.sendTelemetryBatch(t.ShutdownCtx, batch); err != nil {
```
Comment (dselans) [inline services/telemetry/telemetry.go:97]: I might be not seeing it - it looks like we're sending a batch as soon as we receive work. How do we hit the default/noop case?

File: types/types.go [hunk 3]
```diff
@@ -4,4 +4,5 @@ package types
 
 const (
 	PubSubChangesTopic = "changes"
+	UibffEndpoint      = "https://api.dev.streamdal.com"
```
Comment (dselans) [inline types/types.go:7]: We should change this right now so we don't forget to do it later.

**Review by dselans — CHANGES_REQUESTED** [review]
A couple of notes/comments/questions.

File: config/config.go (hunk 1 above)
Comment (blinktag) [reply config/config.go]: Done!

File: services/telemetry/telemetry.go (hunk 2 above)
Comment (blinktag) [reply services/telemetry/telemetry.go]: Yup, old duplicated code, removed

File: types/types.go (hunk 3 above)
Comment (blinktag) [reply types/types.go]: Done!

**Pull request merged**